CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
+- Add: support for APIv2 notifications, which opens up for true APIv2 federation (#2986)
```

Comment:
I think it would be a good idea to include the URL we are now supporting, i.e.:

```
- Add: support for APIv2 notifications (POST /v2/op/notify), which opens up for true APIv2 federation (#2986)
```